server/src/org/apache/cloudstack/network/ssl/CertServiceImpl.java

```diff
@@ -14,7 +14,7 @@
 // KIND, either express or implied.  See the License for the
 // specific language governing permissions and limitations
 // under the License.
-package org.apache.cloudstack.network.lb;
+package org.apache.cloudstack.network.ssl;
```

Comment:
So here is what happened, @rafaelweingartner :
The service definition was in cloud.com... the service implementation was (already) in org.apache.cloudstack...
then I moved the definition around because of your and @resmo 's comments.
 Now I could of course take a few seconds to move this code but I won't right now. I am in the process of building on it and will get to that in a next iteration.